docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -64,7 +66,24 @@ npx @elastic/synthetics .
 [[synthetics-test-suite-send]]
 === Send to {stack}
 
-After testing locally, you can run the test suite using the Elastic Synthetics integration or {heartbeat}
+// Is the push command now the preferred way?
+// If yes, we should definitely include it here and not on its own page.
+// Should it be a tab alongside "Heartbeat" and "Elastic Agent" instead?
```

Comment:
I like the idea of a separate tab to distinguish the running modes.  @drewpost has previously defined the name as _Global managed testing infrastructure_ but that's obviously quite a long tab name.

Remember too that this only works for those in the public beta.  We should make that clear.

I wonder if we should call the tab _Project Monitors_ as that's the distinction here, and is true for both those running on the service, and in the future on self-managed nodes.

Reply:
>Remember too that this only works for those in the public beta. We should make that clear.

How does someone opt into the public beta? Should we include those instructions here?

Reply:
@colleenmcginnis right now they're pointed to a Google Form which gets their account's ID auto-filled so that they can submit a request to be accepted into the beta.

@paulb-elastic I like the idea of project monitors! Short and illustrative name!

My concern with project monitors is that it seems to be in somewhat of a different category than Heartbeat or Elastic Agent. That's because Heartbeat and Elastic Agent are how the monitors are _run_, while "project monitors" refers to how they are _created_.

IMO it might be worth considering changing the names to "Project Monitors", "Inline Heartbeat Monitors" (referring to how they get created and run in Heartbeat), and "Integration Monitors" (referring to how they get created in Kibana and pushed to run in Elastic Agent).

Then on each of those we could start with general information and add a section explaining how monitors get run.

However, I think it should be fine to just add "Project Monitors" alongside "Heartbeat" and "Elastic Agent" just for now so that we keep the scope of these changes minimal, unless folks do think that change is worth making earlier.

Also, just as a note (I think I have mentioned this elsewhere too), I think the project monitors page should start with a clear indication that this is our preferred way to do use Synthetics, and explain its advantages.